Handler/DocumentPublishHandler.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AutomationBundle\Handler;
```

Comment:
@wachterjohannes The change of this namespace has two effects:
* References in JS have to be updated: https://github.com/sulu/sulu/pull/3190/files#diff-c528ccfe4ab86c56b1930b8057ea4495L905
* The values in the `au_tasks` table must be updated.